src/main/scala/djinni/JNIMarshal.scala

```diff
@@ -75,7 +75,10 @@ class JNIMarshal(spec: Spec) extends Marshal(spec) {
       case MSet => "Ljava/util/HashSet;"
       case MMap => "Ljava/util/HashMap;"
     }
-    case e: MExtern => e.jni.typeSignature.get
+    case e: MExtern => e.body match {
+      case e: Enum if e.flags => "Ljava/util/EnumSet;"
+      case _ => e.jni.typeSignature
```

Comment:
```suggestion
      case _ => e.jni.typeSignature.get
```

Reply:
I've updated the commit. Didn't realize this changed compared to the dropbox codebase (which does not contain the `.get`)

Reply:
Thanks!